src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Xamarin.ProjectTools.csproj

```diff
@@ -1,4 +1,4 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
BOM removal?

Reply:
Is our `.editorconfig` wrong? That's what caused this:

https://github.com/xamarin/xamarin-android/blob/e4debf72015549e6e19cbfb247a47fead6ce7a01/.editorconfig#L7-L15

We could change `.csproj` to `utf8-bom`, but I don't know what it's supposed to be actually.